Pipfile

```diff
@@ -31,6 +31,7 @@ pyyaml = "*"
 Flask-SQLAlchemy = "*"
 PyMySQL = "*"
 numba = "*"
+cryptography = "*"
```

Comment:
@zmhassan how is this needed in this PR?

Reply:
When I run locally on mac with mysql and fact_store running it fails to work. Unless I install this package.